### PR TITLE
Onboarding task card behavior on hover

### PR DESCRIPTION
**Pull request by rinagobo**
1. Added &.hovered to the style of OnboardingCardContainer in StyledOnboardingCard.js
2. Created highlight function for OnboardingCard component when it's hovered

### CODE REVIEW

File: apps/user-profile/src/components/modules/UserOnboardingModal/OnboardingCard/OnboardingCard.js [hunk 1]
```diff
@@ -12,8 +12,12 @@ export default function OnboardingCard({
     subtitle,
     completed,
 }) {
+    const [isActive, setIsActive] = useState(false);
+    const detectHover = event => {
+        setIsActive(current => !current);
+      };
     return (
-        <OnboardingCardContainer completed = {completed}>
+        <OnboardingCardContainer completed = {completed} className={isActive ? 'hovered' : ''} onMouseEnter={detectHover} onMouseLeave={detectHover}>
```
Comment (judeamos) [inline apps/user-profile/src/components/modules/UserOnboardingModal/OnboardingCard/OnboardingCard.js:20]: For a hover effect, you don't need to actually create the functionality to know whether someone has hovered over an html element/tag. There is a CSS selector you can use to easily do this. have a look at this to see how: https://www.w3schools.com/cssref/sel_hover.php

File: apps/user-profile/src/components/modules/UserOnboardingModal/OnboardingCard/StyledOnboardingCard.js [hunk 2]
```diff
@@ -62,6 +62,10 @@ export const OnboardingCardContainer = styled.div`
     background: ${props => props.completed ? '#E2E2E2' : 'white'};
     box-shadow: ${props => props.completed ? '' : '0px 10px 18px 10px rgba(127, 126, 127, 0.25)'};
     margin-bottom: 15px;
+
+    &.hovered {
```
Comment (judeamos) [inline apps/user-profile/src/components/modules/UserOnboardingModal/OnboardingCard/StyledOnboardingCard.js:66]: Use CSS psuedo selector instead, i've already pasted a link on another comment here

File: apps/user-profile/src/components/modules/UserOnboardingModal/OnboardingCard/OnboardingCard.js [hunk 3]
```diff
@@ -12,8 +12,12 @@ export default function OnboardingCard({
     subtitle,
     completed,
 }) {
+    const [isActive, setIsActive] = useState(false);
```
Comment (judeamos) [inline apps/user-profile/src/components/modules/UserOnboardingModal/OnboardingCard/OnboardingCard.js:15]: you can remove this logic once CSS applied

File: apps/user-profile/src/components/modules/UserOnboardingModal/OnboardingCard/OnboardingCard.js [hunk 4]
```diff
@@ -12,8 +12,12 @@ export default function OnboardingCard({
...
+    const detectHover = event => {
```
Comment (judeamos) [inline apps/user-profile/src/components/modules/UserOnboardingModal/OnboardingCard/OnboardingCard.js:16]: Also not needed

**Review by judeamos — COMMENTED** [review]
Have a look at comments below, let me know once you've made the changes.

File: apps/user-profile/src/components/modules/UserOnboardingModal/OnboardingCard/OnboardingCard.js (hunk 1 above)
Comment (rinagobo) [reply apps/user-profile/src/components/modules/UserOnboardingModal/OnboardingCard/OnboardingCard.js]: Reverted.

File: apps/user-profile/src/components/modules/UserOnboardingModal/OnboardingCard/StyledOnboardingCard.js (hunk 2 above)
Comment (rinagobo) [reply apps/user-profile/src/components/modules/UserOnboardingModal/OnboardingCard/StyledOnboardingCard.js]: Modified code using css hover selector.

File: apps/user-profile/src/components/modules/UserOnboardingModal/OnboardingCard/OnboardingCard.js (hunk 3 above)
Comment (rinagobo) [reply apps/user-profile/src/components/modules/UserOnboardingModal/OnboardingCard/OnboardingCard.js]: Reverted.

File: apps/user-profile/src/components/modules/UserOnboardingModal/OnboardingCard/OnboardingCard.js (hunk 4 above)
Comment (rinagobo) [reply apps/user-profile/src/components/modules/UserOnboardingModal/OnboardingCard/OnboardingCard.js]: Reverted.

**Review by judeamos — APPROVED** [review]
awsome work, looks good.

**Pull request merged**